hq/test/notifications/AnghammaradNotificationsTest.scala

```diff
@@ -1,30 +0,0 @@
-package notifications
-
-import org.scalatest.{FreeSpec, Matchers}
-
-
-class AnghammaradNotificationsTest extends FreeSpec with Matchers {
-  "outdatedCredentialWarning" - {
-    "TODO" ignore {}
-  }
-
-  "passwordWithoutMfaWarning" - {
-    "TODO" ignore {}
-  }
-
-  "outdatedCredentialFinalWarning" - {
-    "TODO" ignore {}
-  }
-
-  "passwordWithoutMfaFinalWarning" - {
-    "TODO" ignore {}
-  }
-
-  "outdatedCredentialRemediation" - {
```

Comment:
Oh I guess we missed these tests when we wrote the notifications? What is a useful thing to test here I wonder? 🤔

Reply:
There wasn't anything useful to test about the message strings, so these tests were never completed. I think it's fine to delete this empty test file.